src/nomad_measurements/ppms/__init__.py

```diff
@@ -5,19 +5,57 @@
 from pydantic import Field
 
 
-class DataParserEntryPoint(ParserEntryPoint):
+class DataParserEntryPointETO(ParserEntryPoint):
+    parameter: int = Field(0, description='Custom configuration parameter')
+
+    def load(self):
+        from nomad_measurements.ppms.parser import PPMSETOParser
+
+        return PPMSETOParser(**self.dict())
+
+
+eto_parser = DataParserEntryPointETO(
+    name='DataParser',
+    description='New parser entry point configuration.',
+    mainfile_name_re=r'.+\.dat',
+    mainfile_mime_re='text/plain|application/x-wine-extension-ini',
+    mainfile_contents_re=r'BYAPP, Electrical Transport Option',
+)
+
+
+class DataParserEntryPointACT(ParserEntryPoint):
+    parameter: int = Field(0, description='Custom configuration parameter')
+
+    def load(self):
+        from nomad_measurements.ppms.parser import PPMSACTParser
+
+        return PPMSACTParser(**self.dict())
+
+
+act_parser = DataParserEntryPointACT(
+    name='DataParser',
+    description='New parser entry point configuration.',
```

Comment:
please add description in each of these parser entry point, otherwise it is hard to see what are they parsing